addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -184,8 +184,10 @@ export class Powerbox {
         const showOnceOnKeyup = () => {
             this.show();
             openOnKeyupTarget.removeEventListener('keyup', showOnceOnKeyup, true);
-            initialTarget = openOnKeyupTarget;
-            this._initialValue = openOnKeyupTarget.textContent;
+            const selection = this.options.document.getSelection();
+            const currentBlock = closestBlock(selection.anchorNode) || this.options.editable;
```

Comment:
This should probably check that `selection` exists before getting `anchorNode` from it. From what I can see, `closestBlock` should return `undefined` if `selection.anchorNode` is not defined so that's ok.

```suggestion
            const currentBlock = (selection && closestBlock(selection.anchorNode)) || this.options.editable;
```